src/dc_receive_imf.rs

```diff
@@ -42,6 +42,17 @@ enum CreateEvent {
     IncomingMsg,
 }
 
+/// Return type of dc_receive_imf() and dc_receive_imf_inner().
```

Comment:
This comment tells the same thing as written in the code. I can grep the codebase to see which functions return this, and of course if it is returned it contains the data that the caller needs.

This goes to public documentation of the crate, and `dc_receive_imf_inner` is not even a part of the public API.

Not sure how to make an useful comment because I did not understand what is this when I read the structure definition, maybe something like "Common properties for a batch of chat messages added on reception of a single MIME message".

Reply:
Hm. With "Contains exactly the information about the received message that's useful for its callers to know" I meant that it was made specifically to contain exactly this information - and not more, I mean, `dc_receive_imf()` could return `Vec<Message>` and this would also contain the necessary information but also lots of unnecessary information.

"Common properties for a batch of chat messages added on reception of a single MIME message" sounds hard to understand, but maybe we can write something like:

```
This is the struct that is returned after receiving one email (aka MIME message).

One email with multiple attachments can end up as multiple chat messages, but they all have the same chat_id, state and sort_timestamp.
```